### PR TITLE
Version 299

### PR DESCRIPTION
**Pull request by madmongo1**

### CODE REVIEW

File: example/http/client/crawl/http_crawl.cpp [hunk 1]
```diff
@@ -164,8 +163,9 @@ class worker : public std::enable_shared_from_this<worker>
         crawl_report& report,
         net::io_context& ioc)
```
Comment (vinniefalco) [inline example/http/client/crawl/http_crawl.cpp:164]: I think this parameter should be `net::io_context::executor_type` but we don't have to hold up the release for it.

Comment (madmongo1) [reply example/http/client/crawl/http_crawl.cpp:164]: I did actually consider this. We probably need to agree on a policy. I'm personally ok with encapsulating intent in executors anywhere other than `main()`

Comment (vinniefalco) [reply example/http/client/crawl/http_crawl.cpp:164]: I think the policy is that we should avoid passing execution contexts.

**Comment by vinniefalco** [comment]
Accepted, merge when ready

**Pull request merged**